endpoint/http/src/main/java/de/fraunhofer/iosb/ilt/faaast/service/endpoint/http/request/mapper/AbstractRequestMapper.java

```diff
@@ -156,6 +159,59 @@ protected <T> T parseBody(HttpRequest httpRequest, Class<T> type) throws Invalid
     }
 
 
+    /**
+     * Deserializes HTTP body multipart form data.
+     *
+     * @param httpRequest HTTP request
+     * @param contentType the multipart contentType containing the boundary
+     * @return deserialized payload
+     * @throws InvalidRequestException if deserialization fails
+     * @throws IllegalArgumentException if httpRequest is null
+     */
+    protected Map<String, BlobValue> parseMultiPartBody(HttpRequest httpRequest, ContentType contentType) throws InvalidRequestException {
+        Ensure.requireNonNull(httpRequest, "httpRequest must be non-null");
+        Map<String, BlobValue> map = new HashMap<String, BlobValue>();
```

Comment:
What is the motivation to use BlobValue here? Is this really a BlobValue or are we just using this class because it somehow fits what we need?

Reply:
replaced with TypedInMemoryFile